crates/toml/src/de.rs

```diff
@@ -65,7 +65,11 @@ impl Error {
         self.inner.message()
     }
 
-    /// The start/end index into the original document where the error occurred
+    /// The start/end index (byte offset) into the original document where the error occurred.
+    ///
+    /// If you deserialize from a `&str`, you should (nearly) always get a span. Any missing span is
+    /// more likely to be a bug. If you deserialize from a Document, all span information is lost
+    /// and this method will always return `None`.
```

Comment:
What I said to you was not how I would word things for a comment like this.  In general, I'm not sure I feel inclined to specify this at the moment